tools/pythonpkg/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include_directories(src)
+
+find_package(PythonLibs REQUIRED)
+
+include_directories(.eggs/pybind11-2.6.1-py3.6.egg/pybind11/include/)
```

Comment:
this include dir is unlikely to work on someone else's computer

Reply:
Good catch, I thought I had it as a find_package already